homeassistant/components/image_processing/seven_segments.py

```diff
@@ -69,7 +69,8 @@ def __init__(self, hass, camera_entity, config, name):
                 split_entity_id(camera_entity)[1])
         self._state = None
 
-        self.filepath = os.path.join(self.hass.config.config_dir, 'ocr.png')
+        self.filepath = os.path.join(self.hass.config.config_dir, 'ssocr-' +
+                                     self._name.replace(' ', '_') + '.png')
```

Comment:
Use string formatting please.

Reply:
I've made an edit to use string formatting in the commit that follows this one.
```suggestion
                         'ssocr-{0}.png'.format(
                                self._name.replace(' ', '_')))
```